saleor/product/migrations/0121_productchannellisting.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.0.6 on 2020-06-23 07:43
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def migrate_products_publishable_data(apps, schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+    Product = apps.get_model("product", "Product")
+    ProductChannelListing = apps.get_model("product", "ProductChannelListing")
+
+    if Product.objects.exists():
```

Comment:
I think this check is not needed, if there are no products then the iterator below would not run. One query less #performance ;)